docs/game_sdk/Store.md

```diff
@@ -336,21 +336,39 @@ Fires when the connected user loses an entitlement, either by expiration, revoca
 
 The following are HTTP requests, and should be handled by your game server, rather than a client. They require a Bearer token for an authorization header. This token should be a token of a developer who is authorized to use Dispatch for your application. To get this token, your backend service should go through the [Client Credentials OAuth2 Grant](#DOCS_TOPICS_OAUTH2/client-credentials-grant) and request the scope `applications.entitlements`.
 
+Note that parameters with a `?` after the name denote optional fields. Parameters with a `?` before their type denote nullable fields.
+
+## HTTP-Specific Data Models
```

Comment:
can you make a warning about the deprecated behavior with no limit + user_id specified, and then mark March 1 as the day we're removing the deprecated behavior?